### ISSUE TITLE
NTR: Penile discharge, abnormal  (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7965]
New term request Penile discharge, abnormal  (seidel_190409151657): Penis  UMLS: 0232861

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000108
- label: Abnormal penile discharge
- parents: [Abnormal penis morphology (HP:0000036)](https://hpo.jax.org/app/browse/term/HP:0000036)
- pmid: 
- def: Fluid that originates from the urethra that is not urine, pre-ejaculate or semen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7965
- synonyms: Abnormal urethral discharge(NONE)
